dataland-frontend/tests/e2e/specs/quality-assurance/CheckQaProcedure.ts

```diff
@@ -94,6 +94,7 @@ function testSubmittedDatasetIsInReviewListAndAcceptIt(storedCompany: StoredComp
   cy.get('[data-test="qa-review-section"] .p-datatable-tbody tr').last().click();
 
   cy.get('[data-test="qaRejectButton"').should("exist");
+  cy.get("span[class=p-inputswitch-slider]").should("not.exist");
```

Comment:
It would be great if we could also test for a field that exists but is not visible here to prove that the correct value of the new prop `hideEmptyFields` is set for reviewers.